src/System.Net.Http.Formatting/UriExtensions.cs

```diff
@@ -18,21 +18,12 @@ namespace System.Net.Http
     [EditorBrowsable(EditorBrowsableState.Never)]
     public static class UriExtensions
     {
-#if NETFX_CORE
-        /// <summary>
-        /// Parses the query portion of the specified <see cref="Uri"/>.
-        /// </summary>
-        /// <param name="address">The <see cref="Uri"/> instance from which to read.</param>
-        /// <returns>A <see cref="HttpValueCollection"/> containing the parsed result.</returns>
-        public static HttpValueCollection ParseQueryString(this Uri address)
```

Comment:
Public API change?

Reply:
Technically yes though `HttpValueCollection` now consistently derives from `NameValueCollection` in all of the Formatting assemblies and the returned `object` is actually unchanged. The same is true for `FormDataCollection.ReadAsNameValueCollection()` and `HttpContentFormDataExtensions.ReadAsFormDataAsync(...)` (two overloads) due to the removed `NETFX_CORE`-specific `using` removal in each class. Further, the surface area of `HttpValueCollection` is a touch different in the `netstandard1.3` assembly because it has a base class instead of implementing an interface directly and `Add(...)` is now `virtual`.

> These TFM changes are breaking. Will this be a new major version?

Due to the large number of changes in dependencies for different TFMs in the Formatting package, the major increase in surface area of the `netstandard1.3` assembly in there, and the changes noted here, I lean toward bumping the Formatting package's version to 6.0.0 but leaving other packages at 3.3.0 and 5.3.0 because the main changes elsewhere were to the Owin and Newtonsoft package version bumps.

Thoughts❔

Reply:
Sounds good.